File: Human.py
class Human:

    born_year = 1987

    def say_hello(self):
        print("Hello")

    def say_hello_user(self, user_name):
        print("Hello "+ user_name)



human_one = Human()
print(human_one)
print(human_one.born_year)
human_one.say_hello()#aqui estamos invocando
human_one.say_hello_user("Cesar")#aqui estamos invocando

File: Java/classes/Human.java
package classes;

public class Human{

        //Attributes / Properties. en java los atributos y los metodos hay que definir el tipo
        int born_year = 1987;

        public void say_hello(){ //Estos son mis metodos
            System.out.println(" Hello ");

        }

        public String Say_hello_user(String user_name){
            return "Hello " + user_name;

        }

        public int born_year(int year){
            return year;

        }
        
    }
